static/src/app/library/library.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import leaflet from 'leaflet';
```

Comment:
If you want to see the prototype in action, Matt said to do this in the prototype directory:
`sudo php -S localhost:8090`
That will serve the prototype with javascript so everything looks as it should